keyboards/keebio/nyquist/keymaps/winterNebs/keymap.c

```diff
@@ -0,0 +1,153 @@
+  _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, KC_PGUP, KC_UP,	 KC_PGDN, \
+  _______, _______, KC_PSCR, _______, _______, _______, _______, _______, _______, _______, _______, _______\
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * | Reset|RGB Tg|RGB Md|Hue Up|Hue Dn|      |      |      |      | ScrL | ScrR |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |Sat Up|Sat Dn|Val Up|Val Dn|AGnorm|AGswap| Home |  M1  |  ^   |  M2  |ScrUp |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Qwerty|WORKMN|      | End  |  <   |  v   |  >   |Scrdn |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      | Left | Down | Up   |Right | Vol+ |Pause |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | NKRO |      |      |      |      |             |      |      | Last | Vol- | Next |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] =  LAYOUT( \
+  RESET,   RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, _______, _______, _______, _______, KC_WH_L, KC_WH_R, _______,\
+  RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, AG_NORM, AG_SWAP, KC_HOME, KC_BTN1, KC_MS_U, KC_BTN2, KC_WH_U, _______, \
+  _______, _______, _______, QWERTY,  WORKMAN, _______, KC_END,  KC_MS_L, KC_MS_D, KC_MS_R, KC_WH_D, _______,\
+  _______, _______, _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, KC_VOLU, KC_MPLY, \
+  MAGIC_TOGGLE_NKRO, _______, _______, _______, _______, _______, _______, _______, _______, KC_MPRV, KC_VOLD, KC_MNXT \
+)
+
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
+      break;
+    case WORKMAN:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_WORKMAN);
+      }
+      return false;
+      break;
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+      } else {
+        layer_off(_LOWER);
+      }
+      return false;
+      break;
+    case ADJUST:
+      if (record->event.pressed) {
+        layer_on(_ADJUST);
+      } else {
+        layer_off(_ADJUST);
+      }
+      return false;
+      break;
```

Comment:
These are equivalent to just aliasing the native `MO()` keycodes.

You can do:

```suggestion
```

(1/3)